app/views/media_objects/_resource_description.html.erb

```diff
@@ -145,6 +145,15 @@ Unless required by applicable law or agreed to in writing, software distributed
                                 dropdown_field: :note_type,
                                 dropdown_options: ModsDocument::NOTE_TYPES }} %>
 
+  <div class="form-group">
+    <input id="agreement_checkbox" type="checkbox" checked="false" name="deposit_agreement">
+      I have read and do agree to the
+      <a target="_blank" href="https://era.library.ualberta.ca/agreement/">
+	University of Alberta ERA Audio + Video Deposit and Distribution Agreement
+      </a>
+    </input><br />
```

Comment:
This line is red when viewed on github. I think `<input>` doesn't need a closing tag.